frontend/packages/topology/src/components/list-view/TopologyListViewKindGroup.tsx

```diff
@@ -32,21 +34,21 @@ const TopologyListViewKindGroup: React.FC<TopologyListViewKindGroupProps> = ({
   childNodes.sort((a, b) => a.getLabel().localeCompare(b.getLabel()));
 
   return (
-    <DataListItem key={kind} aria-labelledby={`${kind}_label`} isExpanded>
+    <DataListItem key={kind} aria-label={`${kind}_label`} isExpanded>
       <DataListItemRow className="odc-topology-list-view__kind-row">
         <DataListItemCells
           dataListCells={[
             <DataListCell
               key={kind}
               className="odc-topology-list-view__kind-label"
-              id={`${kind}_label`}
+              id={`${idx}_${kind}_label`}
             >
               {resourceLabel}
             </DataListCell>,
           ]}
         />
       </DataListItemRow>
-      <DataListContent aria-label={kind} id={kind} isHidden={false}>
+      <DataListContent aria-label={`${kind}-${idx}`} id={`${kind}-${idx}`} isHidden={false}>
```

Comment:
Similar issue here. We should not use IDX for a label here. It should be a label or refer a dom element. "Deployment 3" would be read out to use our users, that's not really helpful.